### DISCUSSION TITLE
Seeing Errors with magma@mobilityd.service on Development Environment

### DISCUSSION THREAD

**Discussion by shalinjshah18** [magma/magma#7078] [Q&A]
Hi, I am setting up development environment on Debian machine for Magma AGW.

Followed below steps to build Magma AGW

HOST [magma/lte/gateway]$ vagrant ssh magma
MAGMA-VM [/home/vagrant]$ cd magma/lte/gateway
MAGMA-VM [/home/vagrant/magma/lte/gateway]$ make run

But after that when I review status of all Magma services

I see error for mobilityd.service

● magma@mobilityd.service - Magma mobilityd service
   Loaded: loaded (/etc/systemd/system/magma@mobilityd.service; disabled; vendor preset: enabled)
   Active: active (running) since Mon 2021-05-24 07:34:56 UTC; 48s ago
 Main PID: 6290 (python3)
    Tasks: 13 (limit: 4915)
   Memory: 77.2M (limit: 300.0M)
   CGroup: /system.slice/system-magma.slice/magma@mobilityd.service
           └─6290 python3 -m magma.mobilityd.main

May 24 07:35:05 magma-dev mobilityd[6290]:   import cryptography
May 24 07:35:05 magma-dev mobilityd[6290]: INFO:root:Using GW info: {2: [('10.0.2.2', 'eth0', True)], 'default': {2: ('10.0.2.2', 'eth0')}}
May 24 07:35:05 magma-dev mobilityd[6290]: INFO:root:GW update: GW IP[NO_VLAN]: 10.0.2.2
May 24 07:35:05 magma-dev mobilityd[6290]: INFO:root:Using allocator: 0 static ip: False multi_apn False
May 24 07:35:05 magma-dev mobilityd[6290]: INFO:root:Adding IPv4 block
May 24 07:35:06 magma-dev mobilityd[6290]: INFO:root:Added block 192.168.128.0/24 to the IPv4 address pool
May 24 07:35:06 magma-dev mobilityd[6290]: INFO:root:Adding IPv6 block
**May 24 07:35:06 magma-dev mobilityd[6290]: ERROR:root:Empty IP block**
May 24 07:35:06 magma-dev mobilityd[6290]: INFO:root:Starting mobilityd...
May 24 07:35:06 magma-dev mobilityd[6290]: INFO:root:Listening on address 127.0.0.1:60051

Need help to resolve this.

**Comment by pshelar** [comment]
@shalinjshah18 you can ignore this error if you see IPv4 ip block. The error is about IPv6 ip block.
